pkg/oras/pull_opts.go

```diff
@@ -23,7 +23,7 @@ type PullOpt func(o *pullOpts) error
 
 func pullOptsDefaults() *pullOpts {
 	return &pullOpts{
-		dispatch:   images.Dispatch,
+		dispatch:   images.Walk,
```

Comment:
We should remain using `images.Dispatch` since it dispatches in parallel (i.e. download in parallel) while `images.Walk` is synchronous.

Reply:
done. had to modify signature